arch/lkl/include/uapi/asm/host_ops.h

```diff
@@ -159,6 +162,8 @@ struct lkl_host_operations {
 
 	void (*jmp_buf_set)(struct lkl_jmp_buf *jmpb, void (*f)(void));
 	void (*jmp_buf_longjmp)(struct lkl_jmp_buf *jmpb, int val);
+
+	unsigned int (*cpuinfo_get_cpu_cores)(void);
```

Comment:
I don't think this is the right interface.  We should provide a hook for the host environment to provide a fully-populated cpuinfo string.  With this interface, other uses will need to keep adding more and more info hooks for individual fields in this text.  

```suggestion
	const char *(*cpuinfo_get)(void);
```

Reply:
Done.